_sass/_custom.scss

```diff
@@ -9,16 +9,36 @@ p {
     text-align: justify;
 }
 
+/* let portraits grow up to their max size */
 img.portait {
-    width: 150px;
+    display: block;
+    width: 80%;
     border-radius: 50%;
+    margin-top: 3rem;
+    max-width: 300px;
```

Comment:
300px is quite a lot, 200px or 220px?

Reply:
oh, I just looked at Tim's face which is 300 by 300, the others are smaller - so yes, 200 seems okay